src/components/Footer/Footer.scss

```
@@ -1 +1,89 @@
 @import '../../styles/common.scss';
+
+.footer {
+  @include flexCenter;
+  width: 100%;
+  background-color: $bg-gray;
+  min-width: 1050px;
+  padding-right: 20px;
+  padding-left: 20px;
```

```suggestion
padding: 0 20px;
```
단축속성이용해서 위와 같이 표현도 가능합니다!